app/styles/components/_menu-chevron.scss

```diff
@@ -0,0 +1,6 @@
+.wds-menu-chevron {
+	float: right;
+	margin-top: 5px;
+	transform: rotate(-90deg);
```

Comment:
it's not obvious how to use this component. Probably we need a page with some examples